repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/ProjectConfigurationRepositoryTestImpl.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.pipeline.datastorage.dao.repository
+import cromwell.pipeline.datastorage.dto.{ ProjectConfiguration, ProjectConfigurationId, ProjectId }
+
+import scala.collection.mutable
+import scala.concurrent.Future
+
+class ProjectConfigurationRepositoryTestImpl extends ProjectConfigurationRepository {
+
+  private val configurations: mutable.Map[ProjectConfigurationId, ProjectConfiguration] = mutable.Map.empty
+
+  def addConfiguration(projectConfiguration: ProjectConfiguration): Future[Unit] = {
+    configurations += (projectConfiguration.id -> projectConfiguration)
+    Future.successful(())
```

Comment:
```suggestion
    Future.unit
```
Same for `updateConfiguration`

Reply:
@myazinn done